README.md

```diff
@@ -57,6 +57,7 @@ Target problems are very expressive and thus work well as targets for reduction,
 - Entailment in Intuitionistic Multiplicative Sub-Exponential Linear Logic (`IMSELL_cf_PROVABILITY3` in [`ILL/IMSELL.v`](theories/ILL/IMSELL.v))
 - Provability in Hilbert-style calculi (`HSC_PRV` in [`HilbertCalculi/HSC.v`](theories/HilbertCalculi/HSC.v))
 - Recognizing axiomatizations of Hilbert-style calculi (`HSC_AX` in [`HilbertCalculi/HSC.v`](theories/HilbertCalculi/HSC.v))
+- Separation Logic ([SeparationLogic/seplogic.v](theories/SeparationLogic/seplogic.v) and [SeparationLogic/min_seplogic.v](theories/SeparationLogic/min_seplogic.v))
```

Comment:
The points in the list are supposed to be decision problems. `Separation Logic` is not a decision problem. Maybe `Satisfiability in Separation Logic`?

Reply:
will fix